apps/github-cascading-app/src/cascading/cascading-probot.ts

```diff
@@ -51,19 +51,21 @@ export class CascadingProbot extends Cascading {
       }).catch(() => null)));
 
     const configFileValidResponses = configFileResponses
-      .filter((res) => res && Array.isArray(res.data) && res.data.length > 0);
+      .map((res) => res && !Array.isArray(res.data) && res.data || null)
+      .filter((res) => !!res);
 
     const configFileResponse = configFileValidResponses[0];
 
     if (configFileValidResponses.length > 1) {
-      this.logger.warn(`Several configuration files have been found (${configFileValidResponses.map((c) => c && Array.isArray(c.data) && c.data[0].path).join(', ')}). The files will be ignored.`);
+      this.logger.warn(`Several configuration files have been found (${configFileValidResponses.map((c) => c?.path).join(', ')}). The files will be ignored.`);
     } else if (!configFileResponse) {
       this.logger.warn('No remote Configuration found, the default configuration will be used');
     } else {
       try {
-        const parsedConfig = Array.isArray(configFileResponse.data) && configFileResponse.data[0].content && JSON.parse(configFileResponse.data[0].content);
+        const configFileResponseWithContent: { content?: string; encoding?: 'utf8' | 'utf-8' | 'base64' } = configFileResponse as unknown as any;
```

Comment:
What's the difference here between as unknown as any and just as any ?

Reply:
`as any` was rising an exception at transpile time so I added the `as unknown` to make it passed (not sure why the `as any` was rising an exception here :S)